air_quality_management/myapp/views.py

```diff
@@ -158,3 +161,62 @@ class ChangePasswordView(SuccessMessageMixin, PasswordChangeView):
     template_name = 'user/change_password.html'
     success_message = _('Successfully Changed Your Password')
     success_url = reverse_lazy('users-profile')
+
+
+def pollution_level(air_quality_index):
```

Comment:
tên private method theo convention sau nhé
```suggestion
def _pollution_level(air_quality_index):
```

Reply:
resolved